src/main/java/karuna/karuna_backend/content/infrastructure/ContentController.java

```diff
@@ -31,4 +36,14 @@ ContentDTO getContentByPage(@PathVariable String page) {
         return contentService.getContentByPage(page);
     }
 
+
+    @Operation(summary = "Add or update Content", description = "Add Content or update it if row with same page and key exist.")
+    @ApiResponse(
+            responseCode = "201",
```

Comment:
In api response u have 201, but endpoint returns 200, to change it u should use @ResponseStatus, but its think to discuss, we make here create (so should be 201), but we make update too (shoudl be 200?) @BarSwi what u think? I think 200 will be better, but we can have to response codes, one for create and one for update, im not sure, probably 200 its enough